src/beam.cpp

```diff
@@ -1058,14 +1058,17 @@ void BeamSegment::CalcBeamStemLength(Staff *staff, data_BEAMPLACE place, bool is
         }
     }
 
+    short minDuration = DUR_4;
```

Comment:
Suggest use int.   coord->m_dur is type int.

Reply:
I ended up missing one place. Thanks.